contracts/actions/OperationRunner.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity >=0.7.6;
+pragma abicoder v2;
+
+import "../interfaces/mcd/IJoin.sol";
+import "../interfaces/mcd/IManager.sol";
+import "./ActionBase.sol";
+import "hardhat/console.sol";
+import "../ServiceRegistry.sol";
+
+struct Operation {
+  string name;
+  bytes[][] callData;
+  bytes32[] actionIds;
+  address serviceRegistryAddr;
```

Comment:
serviceRegistryAddress can be immutable instead I guess it will be allways the same